tests/integration/acp/nac/setup_then_start/collection_patch_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package test_acp_nac_setup_then_start
+
+import (
+	"testing"
+
+	"github.com/sourcenetwork/immutable"
+
+	"github.com/sourcenetwork/defradb/tests/action"
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+	"github.com/sourcenetwork/defradb/tests/state"
+)
+
+func TestNAC_GatesCollectionPatchPreSetup_AllowIfAuthorizedElseError(t *testing.T) {
```

Comment:
todo: This test does not make a lot of sense to me.  I think it either needs more/better documentation, or it needs to disappear.

Why would adding a GQL schema before restart, affect a NAC permission on an unrelated function after restart?

It feels like this test does not justify it's maintenance cost, and is just going to add to everyone's dev cycles (is another test to wait for, locally and in the CI).

It also tests multiple things (same conversation as an early comment on another test).

Reply:
Explained in the other comment why I added this test, tldr: even if chance of this nac op failing on restart is low I still wanted a restart test, but I will remove it to adhere to this todo.